README.md

```diff
@@ -73,10 +73,10 @@ conda create -n cadquery
 conda activate cadquery
 
 # Install the latest released version
-conda install -c conda-forge cadquery occt=7.7.0
+mamba install -c conda-forge cadquery
```

Comment:
`mamba` is not available in the base environment of all conda distributions, so this may be confusing for those who do not use conda regularly.

Reply:
For those new to conda,  I think it's likely that they are starting without any conda distribution at all.  In that case, the CadQuery installation documentation covers the steps to install Mambaforge which includes mamba.

This PR changes the recommended install method to use mamba.  This change is meant to prevent common cadquery installation issues:

1.  Install failures due to out of memory issues on low mem machines
2. Slow installation (in some cases extremely slow >10x elapsed time than mamba)

Would it help to mention `conda install` as an alternative or include another link to the mamba related docs?

Reply:
I think it is clear, maybe simply mention mambaforge in the Readme too?

Reply:
I've added a link to mambaforge in the README and the note:

``mamba install`` is recommended over ``conda install`` for faster and less memory intensive cadquery installation.